File: app/src/main/java/com/BlizzardArmory/diablo/character/Kills.java
package com.BlizzardArmory.diablo.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.apache.commons.lang3.builder.ToStringBuilder;

public class Kills {

    @SerializedName("elites")
    @Expose
    private Integer elites;

    public Integer getElites() {
        return elites;
    }

    public void setElites(Integer elites) {
        this.elites = elites;
    }

    @Override
    public String toString() {
        return new ToStringBuilder(this).append("elites", elites).toString();
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/diablo3/character/Kills.kt
package com.BlizzardArmory.model.diablo.diablo3.character

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Kills.
 */
@Keep
data class Kills(

    @SerializedName("elites")
    var elites: Int? = null

)